test/units/TEST-13-NSPAWN.machinectl.sh

```diff
@@ -132,13 +132,11 @@ machinectl show-image clone1
 machinectl rename clone1 clone2
 (! machinectl show-image clone1)
 machinectl show-image clone2
-if lsattr -d /var/lib/machines >/dev/null; then
```

Comment:
This was added explicitly to avoid this error:

https://github.com/systemd/systemd/pull/27634#issuecomment-1546366352

https://github.com/systemd/systemd/compare/1f4fe8e1131bd7fa80b9f7621147d4a255737260..bb152e764f8056d1034efeb0f19ba62999cbfcb0

So please drop this commit

Reply:
I have no idea why calling lsattr prevents such error. But OK, the change is dropped now.

Reply:
@mrc0mmand ?

Reply:
The test mounts `tmpfs` on `/var/log/machines` to not pollute the base image, and on older kernels you can't do `chattr()` on `tmpfs`, so `lsattr` is used here to check if it's supported:

```
# lsattr -d /tmp/foo/
lsattr: Inappropriate ioctl for device While reading flags on /tmp/foo/
```

Reply:
It would probably be good to have that as a comment in the script